src/frontend/src/btc/schedulers/btc-wallet.scheduler.ts

```diff
@@ -46,95 +62,153 @@ export class BtcWalletScheduler implements Scheduler<PostMessageDataRequestBtc>
 		});
 	}
 
-	private async loadBtcTransactionsData({ btcAddress }: { btcAddress: BtcAddress }): Promise<{
-		newTransactions: BitcoinTransaction[];
-		latestBitcoinBlockHeight: number | undefined;
-	}> {
+	private async loadBtcTransactionsData({
+		btcAddress
+	}: {
+		btcAddress: BtcAddress;
+	}): Promise<CertifiedData<BtcTransactionUi>[]> {
 		try {
 			const { txs: fetchedTransactions } = await btcAddressData({ btcAddress });
```

Comment:
Not for this PR, just something I noticed: if you want you can "group" the calls here.

```
const [{ txs: fetchedTransactions }, latestBitcoinBlockHeight] = await Promise.all([
  btcAddressData({ btcAddress }),
  btcLatestBlockHeight()
```

Reply:
👍